src/plugincontainer.cpp

```diff
@@ -952,6 +952,12 @@ void PluginContainer::reloadPlugin(QString const& filepath)
 
 void PluginContainer::unloadPlugins()
 {
+  if (m_Organizer) {
+    // clearPlugins() must be called before loadPlugins() since it actually

```

Comment:
I don't understand that comment... This is `unloadPlugins` but this mentions "before `loadPlugins`". Would be nice clarifying the context here.

Reply:
Better?